lib/rucio/web/ui/static/did.js

```diff
@@ -851,6 +851,21 @@ build_page = function() {
             $('#loading').html('<font color="red">Could not find the DID.</font>');
         }
     });
+
+    r.did_get_generic_metadata({
+        'scope': scope,
+        'name': name,
+        success: function(data) {
+            $("#loading").html(""); 
+            var sorted_keys = Object.keys(data).sort();
+            for(var i=0; i<sorted_keys.length; ++i) {
+                $('#t_generic_metadata').append($('<tr><th>' + sorted_keys[i] + '</th><td>' + data[sorted_keys[i]] + '</td></tr>'));
+            }
+        },
+        error: function(jqXHR, textStatus, errorThrown) {
+            $('#generic_metadata').hide();
```

Comment:
@tbeerman  I am hiding html if the rest api returns an error.

Reply:
Please do it the other way around: hide the whole div by default (`style="visibility: hidden;"`) and only make it visible when the call was successful.